csharp/src/Google.Protobuf.Test/Google.Protobuf.Test.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net451;netcoreapp2.1</TargetFrameworks>
```

Comment:
Was this intentional?

Reply:
VS only shows tests in the explorer for one target and my netcoreapp2.1 tests weren't showing up. It's a temp thing